cmd.js

```diff
@@ -1,19 +1,43 @@
 #!/usr/bin/env node
 
+// Core dependencies
 var fs = require('fs');
 var path = require('path');
+
+// NPM dependencies
 var detectIndent = require('detect-indent');
+var sortJson = require('./');
+
+// Get all the files
+var files = process.argv.slice(2);
+
+files.forEach(readEachFile);
 
-var filename = path.resolve(process.argv[2]);
-var file = fs.readFileSync(filename, 'utf8');
+function readEachFile(fileName) {
+  var filePath = path.resolve(fileName);
+  fs.readFile(filePath, 'utf8', readingFile);
 
-// Try to detect the indentation and fall back to two spaces if unable.
-var indent = detectIndent(file).indent || '  ';
+  function readingFile(err, file) {
+    if (err) {
+      console.error(err);
+      process.exit(0);
```

Comment:
You are definitely right. it make more sense to change it to `1` as we are terminating with an error. 
sorry